### PR TITLE
added links to rules

### PR DESCRIPTION
**Pull request by DevoKun**

### CODE REVIEW

**Review by thomasmarlin — COMMENTED** [review]
I like the idea of this change!  Having direct links to confusing stuff makes a lot of sense. This is especially awesome for things like the special Tatooine / Bespin / etc setup. That's very useful.

My concern however is with putting the "Generic Systems Deployment" on every single system. Likewise, having the creature rules on every creature.   I can see some the benefits of having it inside the DB. However, I also get a little worried about bloat in the DB too. For example, we wouldn't put "Battle Rules" on every card that battles, we wouldn't put "Deployment Rules" on every card that deploys.  I'm not sure where to draw the line on that one.

This may make a good discussion in Slack

File: Dark.json [hunk 1]
```diff
@@ -44902,6 +45067,8 @@
         "darkSideIcons": 0,
         "gametext": "Dark:  'Bluff Rules' in effect here.  Light:  'Bluff Rules' in effect here."
       },
+      "rules": [{"name":"Tatooine Systems Deployment", "url":"https://www.starwarsccg.org/location-deployment-specifics-tatooine"}],
+      "rules": [{"name":"Bluff Rules", "url":"https://www.starwarsccg.org/bluff-rules/"}],
```
Comment (thomasmarlin) [inline Dark.json:45071]: Should be merged into 1 array

**Review by thomasmarlin — CHANGES_REQUESTED** [review]

File: Dark.json [hunk 2]
```diff
@@ -3725,6 +3726,7 @@
         "gametext": "May carry 2 passengers. Bantha ability = 1/2. May move as a 'react' only to a battle or Force drain (if within range). Each Tusken Raider at same exterior site is power +1.",
         "lore": "Transport, pack animal. Many breeds of different sizes and colors. Three meters tall. Can go weeks without food or water. Found throughout the galaxy."
       },
+      "rules": [{"name":"Creature Rules", "url":"https://www.starwarsccg.org/creature-rules/"}],
```
Comment (thomasmarlin) [inline Dark.json:3729]: Greg noticed this on Slack. We need to remove these "Creature Rules" entries for Creature Vehicles (Only including them for actual creatures).

File: Light.json [hunk 3]
```diff
@@ -1358,6 +1359,7 @@
         "darkSideIcons": 1,
         "gametext": "Light:  If Luke here or out of play, Force Projection is [Immune to Sense].  Dark:  While Luke here, attrition against you everywhere is +1."
       },
+      "rules": [{"name":"Generic Systems Deployment", "url":"https://www.starwarsccg.org/location-deployment-specifics-general"}],
```
Comment (thomasmarlin) [inline Light.json:1362]: Looks like the "Generic" rules were removed for the Dark cards, but are still present for the light-side.  I'm guessing you are still working on this (this is mostly a note for whoever reviews this later to make sure to check both files)

**Comment by scottrick** [comment]
These rule links are really great and I think could be very handy moving forward.  I have two concerns:
* Is there a process to make sure that as the official rules PDF is updated, these rules pages are also updated?  I worry about them getting out of sync.
* It might be worth aligning the URL format before everything starts using it:
Instead of https://www.starwarsccg.org/creature-rules/, maybe we should have the rules all be something like:  https://www.starwarsccg.org/rules/creatures/.  https://www.starwarsccg.org/rules/ could contain a directory of all available special rules, or an online version of the pdf, or something else.

**Comment by DevoKun** [comment]
> These rule links are really great and I think could be very handy moving forward. I have two concerns:
> 
> * Is there a process to make sure that as the official rules PDF is updated, these rules pages are also updated?  I worry about them getting out of sync.
> * It might be worth aligning the URL format before everything starts using it:
>   Instead of https://www.starwarsccg.org/creature-rules/, maybe we should have the rules all be something like:  https://www.starwarsccg.org/rules/creatures/

These rules URL's are not new website content. They have been on the site for years. The only change here is a quicklink to them.

**Comment by thomasmarlin** [comment]
@DevoKun  - drop a note here once this is ready for re-review again (and conflicts resolved). I think a bunch of these will be great to get in

**Pull request closed without merging**